package.json

```diff
@@ -19,7 +19,7 @@
     "build:utils": "cross-env BABEL_ENV=utils babel packages/utils --extensions .ts --out-dir lib/utils",
     "build:locale": "cross-env BABEL_ENV=utils babel packages/locale --extensions .ts --out-dir lib/locale",
     "build:locale-umd": "node ./build/build-locale.js",
-    "build:theme": "rimraf packages/theme-chalk/lib && gulp build --gulpfile packages/theme-chalk/gulpfile.js && cp-cli packages/theme-chalk/lib lib/theme-chalk && rimraf packages/theme-chalk/lib",
+    "build:theme": "cd packages/theme-chalk && yarn clean && yarn build",
```

Comment:
Question: would this change the current directory which could impact the following jobs? E.g. `scripts/build.sh` we have post build jobs for clean up and bundling.

Reply:
Tested with no issue. I am happy to approve as is.

Reply:
Yes, I tried it before. I've tried this in some of my own projects. This ensures consistency with individual package behavior and more concise.